tools/cmake2meson.py

```diff
@@ -179,13 +250,15 @@ def write_entry(self, outfile, t):
             line = "subdir('" + t.args[0].value + "')"
         elif t.name == 'pkg_search_module' or t.name == 'pkg_search_modules':
             varname = t.args[0].value.lower()
+            #self.dependencies.append(i.value.lower() for i in t.args[1:])
```

Comment:
__[flake8]__
* [E265] block comment should start with '# '

<!-- Comment by [SideCI](https://sideci.com) -->